presto-main-base/src/main/java/com/facebook/presto/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -665,6 +664,16 @@ protected Node visitShowCreateFunction(ShowCreateFunction node, Void context)
                     ordering(ascending("argument_types")));
         }
 
+        private QualifiedName getQualifiedName(ShowCreate node, QualifiedObjectName objectName)
+        {
+            List<Identifier> parts = node.getName().getOriginalParts();
+            Identifier tableName = (parts.size() > 2) ? parts.get(2) : ((parts.size() > 1) ? parts.get(1) : parts.get(0));
+            Identifier schemaName = (parts.size() > 2) ? parts.get(1) : ((parts.size() > 1) ? parts.get(0) : new Identifier(objectName.getSchemaName()));
```

Comment:
```suggestion
            Identifier tableName = getLast(parts);
            Identifier schemaName = parts.size() > 1 ? parts.get(parts.size() - 2) : new Identifier(objectName.getSchemaName());
```

nit: We can obtain the `tableName` and `schemaName` more concisely.

Reply:
So I just traced it, looks like o/p would be dfferent if lets say when `parts.size() > 3`?